dspace/config/spring/api/discovery.xml

```diff
@@ -807,6 +815,83 @@
         <property name="spellCheckEnabled" value="true"/>
     </bean>
 
+    <!--The workflowAdmin configuration settings for discovery -->
+    <bean id="workflowAdminConfiguration"
+        class="org.dspace.discovery.configuration.DiscoveryConfiguration"
+        scope="prototype">
+        <property name="id" value="workflowAdmin" />
+        <!--Which sidebar facets are to be displayed -->
+        <property name="sidebarFacets">
+            <list>
+                <ref bean="searchFilterObjectNamedType" />
+                <ref bean="searchFilterType" />
+                <ref bean="searchFilterIssued" />
+                <ref bean="searchFilterSubmitter" />
+            </list>
+        </property>
+        <!--The search filters which can be used on the discovery search page -->
+        <property name="searchFilters">
+            <list>
+                <ref bean="searchFilterObjectNamedType" />
+                <ref bean="searchFilterType" />
+                <ref bean="searchFilterIssued" />
+                <ref bean="searchFilterSubmitter" />
+            </list>
+        </property>
+        <!--The sort filters for the discovery search-->
+        <property name="searchSortConfiguration">
+            <bean class="org.dspace.discovery.configuration.DiscoverySortConfiguration">
+                <!--<property name="defaultSort" ref="sortDateIssued"/>-->
+                <!--DefaultSortOrder can either be desc or asc (desc is default)-->
+                <property name="defaultSortOrder" value="desc"/>
+                <property name="sortFields">
+                    <list>
+                        <ref bean="sortTitle" />
+                        <ref bean="sortDateIssued" />
+                    </list>
+                </property>
+            </bean>
+        </property>
+        <!--Any default filter queries, these filter queries will be used for all
+            queries done by discovery for this configuration -->
+        <property name="defaultFilterQueries">
+            <list>
+                <!--Only find PoolTask and ClaimedTask -->
+                <value>search.resourcetype:PoolTask OR search.resourcetype:ClaimedTask</value>
```

Comment:
would not be better to index the wrofklowitem directly and just limit this configuration to retrieve the workflowitem instead than tasks? the UI need to list the workflowitem and delete / reset such workflow. A single workflow could result in multiple tasks depending on the workflow steps so there is not necessary a one to one mapping. Having the count in the UI based on tasks instead than workflow could generate confusion in the admin

Reply:
That would be possible as well, but would also impact the UI, and I know that a lot of time has been invested in the UI already
I'd suggest we verify whether there's time left once the UI has been finalized

Reply:
@benbosman and @abollini : At a quick glance, I admit I was also a bit confused by this configuration.  If a WorkflowItem can have multiple Tasks, then is it possible that the WorkflowItem would appear multiple times (under multiple tasks) in this index? Would that cause issues in the UI layer?

All that said, I anticipate this *should be similar* to the existing `workflowConfiguration` in discovery.xml....and that configuration *also uses* `PoolTask OR ClaimedTask`.  So, on a second glance, it seems like that is how we've already been indexing WorkflowItems...so, **it appears this configuration likely is correct as-is**.

See https://github.com/DSpace/DSpace/blob/master/dspace/config/spring/api/discovery.xml#L775

This might be simply an opportunity to add a more descriptive inline comment before these defaultFilterQueries for WorkflowItem based indexes.  It's not immediately clear to me why we are indexing the Tasks themselves, but there must be a reason for it (I'd need to dig in further to better understand why myself)